docs/sql/statements/create-role.rst

```diff
@@ -27,5 +27,14 @@ CrateDB cluster. The newly created role does not have any special privileges,
 and those must be assigned afterwards, for details see the
 :ref:`privileges documentation<administration-privileges>`.
 
+.. NOTE::
+
+    ``ROLE`` is essentially the same as ``USER`` with the difference that a
+    ``ROLE`` **cannot** login to the database, and therefore **cannot** be
+    assigned a password, and can be :ref:`granted <granting_roles>` to another
+    ``USER`` or ``ROLE``. On the other hand. a ``USER`` **can** login to the
+    database and **can** also be assigned a password, but **cannot** be granted
+    to another ``USER`` or ``ROLE``.
+
```

Comment:
Yep, thx for insisting, I was a hesitating because we don't allow altering a role to login (become a user) and vice versa, but I agree it's good to have the duality notion already and clearly mention the diffs.